www/source/shared/_setup_environment.html.md.erb

```diff
@@ -1,24 +1,29 @@
 The `hab` command-line interface (CLI) tool downloads its other components when it needs them, so setup for Habitat is centered on where you want the `hab` CLI installed and getting it integrated into your shell.
 
-1.  Open a terminal window. Change directory to the directory where you want to place the `hab` CLI. The following example creates a directory called `bin` and places it at the root of the home directory:
+1.  Open a terminal window. Change directory to the directory where you want to place the `hab` CLI and copy the `hab` CLI to that directory.
 
-        mkdir -p ~/bin && cd ~/bin
+    If you are running Mac OS X, create a directory called `bin` and place it at the root of the home directory:
 
-2. Copy the `hab` CLI to that directory.
+        mkdir -p ~/bin
+        cp where/you/extracted/hab ~/bin
 
-3. Add the location for the `hab` CLI to your `PATH` environment variable.
+    If you are running Linux, copy the `hab` CLI into the `/bin` directory.
+
+        sudo cp where/you/extracted/hab /bin
+
+2. For Mac OS X, add the location for the `hab` CLI to your `PATH` environment variable.
 
        export PATH=$PATH:~/bin
 
     To keep from typing this in for every terminal window you use, add the above statement to your shell profile.
 
-4. Open a terminal window and run `hab setup`. If your host machine is running Linux, the `hab` CLI must run as root, so we suggest running it as root through the `sudo` command.
+3. On Mac OS X, run `hab setup`. If your host machine is running Linux, we suggest running `sudo hab setup`. The `hab` CLI must run as root to place your [origin keys](/docs/concepts-keys#origin-keys) in the default location for the studio to use them (`/hab/cache/keys`).
```

Comment:
It's because when we call `sudo hab studio enter`, the keys are expected to be in `/hab/cache/keys' not `$HOME/.hab/cache/keys`. I guess we could override with an env var to look there.